src/bg.ts

```diff
@@ -1,44 +1,37 @@
 /* 아 코드 뭐같아서 빨리 고쳐야 하는데.... */
-const urlRegex = "^http(s|):\\/\\/";
+const urlRegex = "^http(s?):\\/\\/";
 
 interface PageBlockRule {
     baseURL: string;
-    articleView: string;
-    searchView: string;
+    articleView: RegExp;
+    searchView: RegExp;
 }
 
 const namuWikiBlockRule: PageBlockRule[] = [{
     baseURL: "namu.wiki",
-    articleView: "/w/",
-    searchView: "/go/",
+    articleView: /w/,
```

Comment:
This should also validate it is a subdirectory of baseURL

currently, it will also trigger if:  
https://namu.wiki/shd/w/jdsfkldsjf